ci/rpm7/main/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM fiware/orion-ci:rpm7-env
```

Comment:
What's the diferent between fiware/orion-ci:rpm7 and fiware/orion-ci:rpm7-env?

Maybe the answer is in the PR itself, but I'm unable to find it...

Reply:
Just to check if I'm understanding correctly...

The "regular" docker built from docker/Dockerfile (and available in fiware/orion) is not suitable for CI as it doesn't containt the required building dependencies (the dependencies are downloaded at building time, but cleaned once the contextBroker binary is built, in order to reduce the, see size https://github.com/telefonicaid/fiware-orion/blob/master/docker/Dockerfile#L63).

Under this ligth, it's ok to have separate Dockerfiles used specifically for CI, as the ones you are including in this PR.

Is my understanding correct?

In that case, it's fine, but I'd suggest to include a comment preamble in the two new Dockerfiles included in this PR in order to explain its purpose, how each one related with the other (e.g. explain that one container is used as base layer for the other), and details about how they are built and made available in the dockerhub registry (I mean, is fiware/orion-ci:rpm7 docker built and pushed to dockerhub each nigth? or any other mechanism?)

Reply:
Refactored in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/aaaea19bab8a775a69cc7727ef8a37025faccf0c.

Reply:
Bottom line is that I miss some documentation about how the CI system works (docker involved, etc.). If the Dockerfile is not a good place to include such documentation, maybe ci/READEM.md would be a good place.

Could you create a first draft of ci/README.md in this PR so we can iterate on it and improve, please?

Reply:
Ok)

Reply:
fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/e9647f964cd31c64d32f84ab6877d03921dcc09b